### DISCUSSION TITLE
Black Loading Screen After Mod Install

### DISCUSSION THREAD

**Discussion by IdrisAli99** [CitiesSkylinesMods/TMPE#1694] [Q&A] [answered]
\

**Comment by krzychu124** [comment, marked as answer]
The only reason why it's not working is you didn't follow installation instructions:
https://github.com/CitiesSkylinesMods/TMPE/wiki/Installing-on-Epic-Games
and a reminder...

![image](https://user-images.githubusercontent.com/19638970/203178860-cd513a9f-0bba-4c3e-92d9-de4668ebbbf0.png)
